docs/how-to-guides/advanced-user-guides/tune-etcd-for-large-installs.md

```diff
@@ -6,7 +6,7 @@ title: Tuning etcd for Large Installations
   <link rel="canonical" href="https://ranchermanager.docs.rancher.com/how-to-guides/advanced-user-guides/tune-etcd-for-large-installs"/>
 </head>
 
-When running larger Rancher installations with 15 or more clusters it is recommended to increase the default keyspace for etcd from the default 2GB. The maximum setting is 8GB and the host should have enough RAM to keep the entire dataset in memory. When increasing this value you should also increase the size of the host. The keyspace size can also be adjusted in smaller installations if you anticipate a high rate of change of pods during the garbage collection interval.
+When running larger Rancher installations it is recommended to increase the default keyspace for etcd from the default 2GB. The maximum setting is 8GB and the host should have enough RAM to keep the entire dataset in memory. When increasing this value you should also increase the size of the host. The keyspace size can also be adjusted in smaller installations if you anticipate a high rate of change of pods during the garbage collection interval.
```

Comment:
Is this just to future-proof the docs? It might still be useful to give the reader a concept of when an install crosses over to being "large"

Reply:
It's to bring docs up to speed with current reality more than future-proofing it: 15 is a very small number these days, nothing that we can consider "large".

I agree with your point and tried to address it in c27dd902c76 by adding a pointer to the installation requirements page, which has tables that define what "large" means.